tests/foreman/cli/test_contentviewfilter.py

```diff
@@ -973,3 +975,87 @@ def test_negative_delete_by_name(self, content_view):
             ContentView.filter.delete(
                 {'content-view-id': content_view['id'], 'name': gen_string('utf8')}
             )
+
+    @pytest.mark.stream
+    @pytest.mark.tier2
+    def test_positive_check_filters_applied(self, target_sat, module_org, content_view):
+        """Ensure the applied filters are indicated and listed correctly in the CVV info.
+
+        :id: ab72af3f-6bee-4aa8-a74a-637fe9b7e34a
+
+        :steps:
+            1. Publish first CVV with no filters applied, assert no applied filters are indicated
+               nor listed.
+            2. Randomly add filters to the CV, publish new CVVs and assert that the applied filters
+               are indicated and the correct filters are listed.
+            3. Randomly remove filters from the CV, publish new CVVs and assert that the applied
+               filters are indicated when at least one filter exists and the correct filters were
+               removed.
+
+        :expectedresults:
+            1. Hammer shows correctly if a CVV has filter(s) applied, no matter the filter type,
+               count, nor order.
+            2. Hammer lists correctly the applied filter(s), no matter the filter type, count
+               nor order.
+        """
+        f_types = ['rpm', 'package_group', 'erratum', 'modulemd', 'docker']
+        filters_applied = []
+
+        # Publish first CVV with no filters applied
+        target_sat.cli.ContentView.publish({'id': content_view['id']})
+        cvv = target_sat.cli.ContentView.info({'id': content_view['id']})['versions'][0]
+        cvv_info = target_sat.cli.ContentView.version_info(
+            {'id': cvv['id'], 'include-applied-filters': 'true'}
+        )
+        assert cvv_info['has-applied-filters'] == 'no'
+        assert 'applied-filters' not in cvv_info
+
+        # Randomly add filters to the CV, assert correct CVV info values
```

Comment:
Well, I hope I can. :grinning: 
One reason was that the `has-applied-filters` flag was already tested per filter type (and per test run) in the API counterpart in #11461 (originally the flag was the only thing planned to be implemented in API). But now, with the applied filters even listed via CLI, I felt I need to test that the filters are listed correctly for each published CV version, no matter the filter content type nor inclusion type. Please note that there can be _many_ filters of different (or same, or any combination) types associated with a single CV version. So, what would be the full matrix? No matter the answer it wouldn't be reasonable to test it full, so I decided to go random, gave it 100+ runs to get some statistic (all passed) and pushed for review.
Still, I can see the point of "missing anything on a given automation run". We could change the `k` on line 1014 to limit the risk, or ensure every filter type is present at least once. The price is the test time, as usual.

Reply:
Since the functionality is tested in other PR, I think the extent and scope of testing the flag `has-applied-filters` covered in this test should be sufficient.